src/main/java/com/lucidworks/spark/util/ConfigurationConstants.java

```diff
@@ -0,0 +1,15 @@
+package com.lucidworks.spark.util;
+
+public class ConfigurationConstants {
```

Comment:
Make this an interface so that classes that refer to these statics can just implements the interface